domain/bakerystorage/service/service.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package service
+
+type State interface {
+	ConfigState
+}
+
+type Service struct {
+	*ConfigService
+}
+
+func NewService(st State) *Service {
+	return &Service{
+		ConfigService: NewConfigService(st),
```

Comment:
What is the purpose of not just having the ConfigService here and doing away with the indirection?

Reply:
My next task will be embedding more thing here

A bit like this

https://github.com/juju/juju/blob/main/domain/access/service/service.go#L185-L198